h2o-py/h2o/frame.py

```diff
@@ -2785,6 +2785,34 @@ def which(self):
         """
         return H2OFrame._expr(expr=ExprNode("which", self))
 
+    def idxmax(self,skipna=False, axis=0):
+        """
```

Comment:
Yep, can make that change. Just did it this way since all other methods follow the same convention.